src/components/Checkout/OrderSummary.js

```diff
@@ -8,23 +8,21 @@ import {
   Text,
   Button,
   Divider,
+  useToast,
 } from '@chakra-ui/react';
 import { useStripe, useElements } from '@stripe/react-stripe-js';
 import { CartContext } from '../../util/context';
 import { colors } from '../../util/constants';
+import { FRONT_END_BASE_URL } from '../../util/config';
```

Comment:
Are you sure we need this? You weren't able to use `useHistory` or `useLocation` to do this?

Reply:
I actually just learned about it and yes i might use that